<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>startExam</class>
 <widget class="QMainWindow" name="startExam">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>360</width>
    <height>530</height>
   </rect>
  </property>
  <property name="minimumSize">
   <size>
    <width>360</width>
    <height>530</height>
   </size>
  </property>
  <property name="mouseTracking">
   <bool>false</bool>
  </property>
  <property name="windowTitle">
   <string>DDOK DDOK</string>
  </property>
  <property name="styleSheet">
   <string notr="true">background-color:rgb(255, 255, 255)</string> <!-- /기존:252, 252, 244-->
  </property>
  <widget class="QWidget" name="centralwidget">
   <property name="minimumSize">
    <size>
     <width>500</width> <!--기존: 360-->
     <height>830</height><!--기존: 530-->
    </size>
   </property>
   <widget class="QLabel" name="label_sn_title">
    <property name="geometry">
     <rect>
      <x>20</x>
      <y>23</y>
      <width>64</width>
      <height>15</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <family>나눔스퀘어라운드 Bold</family>
      <weight>75</weight>
      <bold>true</bold>
     </font>
    </property>
    <property name="text">
     <string>학   번 :</string>
    </property>
   </widget>
   <widget class="QLabel" name="label_en_title">
    <property name="geometry">
     <rect>
      <x>20</x>
      <y>50</y>
      <width>64</width>
      <height>15</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <family>나눔스퀘어라운드 Bold</family>
      <weight>75</weight>
      <bold>true</bold>
     </font>
    </property>
    <property name="text">
     <string>시험번호 :</string>
    </property>
   </widget>
   <widget class="QPushButton" name="finishButton">
    <property name="geometry">
     <rect>
      <x>160</x>
      <y>450</y>
      <width>181</width>
      <height>41</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <family>나눔스퀘어라운드 Bold</family>
      <weight>75</weight>
      <bold>true</bold>
     </font>
    </property>
    <property name="accessibleName">
     <string>finishButton</string>
    </property>
    <property name="styleSheet">
     <string notr="true">background-color:rgb(14, 122, 254)</string><!--255, 230, 153-->
    </property>
    <property name="text">
     <string>시 험 종 료</string>
    </property>
    <property name="autoDefault">
     <bool>true</bool>
    </property>
    <property name="flat">
     <bool>false</bool>
    </property>
   </widget>
   <widget class="QLabel" name="label_sn_set">
    <property name="geometry">
     <rect>
      <x>100</x>
      <y>23</y>
      <width>150</width>
      <height>16</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <family>나눔스퀘어라운드 Bold</family>
      <weight>75</weight>
      <bold>true</bold>
     </font>
    </property>
    <property name="text">
     <string>TextLabel</string>
    </property>
   </widget>
   <widget class="QLabel" name="label_en_set">
    <property name="geometry">
     <rect>
      <x>100</x>
      <y>50</y>
      <width>150</width>
      <height>16</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <family>나눔스퀘어라운드 Bold</family>
      <weight>75</weight>
      <bold>true</bold>
     </font>
    </property>
    <property name="text">
     <string>TextLabel</string>
    </property>
   </widget>
   <widget class="QLabel" name="exam_ongoing_alert">
    <property name="geometry">
     <rect>
      <x>20</x>
      <y>140</y>
      <width>321</width>
      <height>41</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <family>나눔스퀘어_ac Bold</family>
      <pointsize>14</pointsize>
      <weight>75</weight>
      <bold>true</bold>
     </font>
    </property>
    <property name="styleSheet">
     <string notr="true">background-color:rgb(14, 122, 254)</string><!--255, 230, 153-->
    </property>
    <property name="text">
     <string>감시기능 작동 중</string>
    </property>
    <property name="alignment">
     <set>Qt::AlignCenter</set>
    </property>
   </widget>
   <widget class="QTextBrowser" name="log_list">
    <property name="geometry">
     <rect>
      <x>20</x>
      <y>200</y>
      <width>321</width>
      <height>231</height>
     </rect>
    </property>
   </widget>
   <widget class="QLabel" name="label_title_title">
    <property name="geometry">
     <rect>
      <x>20</x>
      <y>77</y>
      <width>71</width>
      <height>16</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <family>나눔스퀘어라운드 Bold</family>
      <weight>75</weight>
      <bold>true</bold>
     </font>
    </property>
    <property name="text">
     <string>시험제목 :</string>
    </property>
   </widget>
   <widget class="QLabel" name="label_title_set">
    <property name="geometry">
     <rect>
      <x>100</x>
      <y>77</y>
      <width>240</width>
      <height>16</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <family>나눔스퀘어라운드 Bold</family>
      <weight>75</weight>
      <bold>true</bold>
     </font>
    </property>
    <property name="text">
     <string>TextLabel</string>
    </property>
   </widget>
   <widget class="QLabel" name="label_time_set">
    <property name="geometry">
     <rect>
      <x>100</x>
      <y>104</y>
      <width>240</width>
      <height>16</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <family>나눔스퀘어라운드 Bold</family>
      <weight>75</weight>
      <bold>true</bold>
     </font>
    </property>
    <property name="text">
     <string>TextLabel</string>
    </property>
   </widget>
   <widget class="QLabel" name="label_time_title">
    <property name="geometry">
     <rect>
      <x>20</x>
      <y>104</y>
      <width>71</width>
      <height>16</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <family>나눔스퀘어라운드 Bold</family>
      <weight>75</weight>
      <bold>true</bold>
     </font>
    </property>
    <property name="text">
     <string>시험시간 :</string>
    </property>
   </widget>
  </widget>
  <widget class="QStatusBar" name="statusbar"/>
 </widget>
 <resources/>
 <connections/>
</ui>
